src/plugins/intel_cpu/src/shape_inference/custom/strided_slice.cpp

```diff
@@ -56,20 +56,34 @@ Result StridedSliceShapeInfer::infer(const std::vector<std::reference_wrapper<co
         }
     };
 
-    for (size_t in_idx = 0, out_idx = 0; in_idx < shapeIn.size(); ++in_idx) {
-        if (m_new_axis_mask_set.count(in_idx)) {
-            // deal with new_axis_mask
-            m_outputShape[out_idx] = 1;
+    const auto shapeInSize = shapeIn.size();
+    const auto newAxisMaskSize = m_new_axis_mask_set.size();
+    const auto maxAxisSize = shapeInSize + newAxisMaskSize;
+    const auto outputShapeSize = m_outputShape.size();
+    bool newAxis = false;
+    bool shrinkAxis = false;
```

Comment:
seems `newAxis` `shrinkAxis` are local inside the loop, these declarations can be moved into line 71 & 72

Reply:
because we use this custom shape infer to replace Ngraphshapeinfer in CPU plugin, so we hope it is as quickly as possible, so I put declarations here to reduce stack memory allocation for newAxis and shrinkAxis, maybe gcc have good optimization for local instance inside loop, but I still think put declarations at the beginning is better.